extensions/crc/src/extension.ts

```diff
@@ -66,8 +63,12 @@ export async function activate(extensionContext: extensionApi.ExtensionContext):
     crcStatus = await commander.status();
   } catch (err) {
     console.error('error in CRC extension', err);
+    crcStatus = { CrcStatus: 'Unknown', Preset: 'Unknown' };
```

Comment:
```suggestion
    crcStatus = unknownStatus
```

See other command about moving const top top of file